client/src/App.jsx

```diff
@@ -31,57 +31,48 @@ export const App = () => {
   return (
     <ThemeSelector>
       <div>
-        <Row id="topRow" style={{ marginTop: '10px' }}>
-          <Col md="4"></Col>
-          <Col md="4" style={{ textAlign: 'center' }}>
-            <a href="/">
-              <img src="/IQEngine.svg" alt="IQEngine" />
-            </a>
-          </Col>
-          <Col md="4">
-            <Row>
-              <Container className="nav justify-content-end">
-                <Navbar variant="dark" style={{ paddingTop: '0px', paddingBottom: '0px' }}>
-                  <Nav.Link href="/about" style={{ fontSize: '24px', paddingLeft: '0px', paddingRight: '25px' }}>
-                    About
-                  </Nav.Link>
-
-                  <Nav.Link href="/sigmf" style={{ fontSize: '24px', paddingLeft: '0px', paddingRight: '25px' }}>
-                    SigMF
-                  </Nav.Link>
+        <div className="container flex mb-40" id="topRow">
+          <a href="/" className="absolute mt-4 pt-2 top-0 left-1/2 transform -translate-x-1/2">
+            <img src="/IQEngine.svg" alt="IQEngine" />
+          </a>
 
-                  <Nav.Link href="/plugins" style={{ fontSize: '24px', paddingLeft: '0px', paddingRight: '10px' }}>
-                    Plugins
-                  </Nav.Link>
-                </Navbar>
-              </Container>
-            </Row>
-            <Row className="mb0 mt0">
-              <Container className="nav justify-content-end">
-                <a
-                  className="nav-link"
-                  style={{ display: 'inline-flex', fontSize: '24px', paddingLeft: '0px', paddingRight: '20px' }}
-                  target="_blank"
-                  rel="noreferrer"
-                  href="https://discord.gg/k7C8kp3b76"
-                >
-                  <img src="/discord.svg" style={{ width: '22px' }} alt="Discord" />
-                  Discord
+          <div className="absolute right-0 py-2 mr-2">
```

Comment:
Is it worth putting these into their own 'Header.tsx' file?

Reply:
yep good point

Reply:
looks like we already have navbar and footer components but they are only used for the v2 version, pretty soon we'll merge the two versions into one and then we can reorganized this stuff